Microsoft.Toolkit.Services/Services/Weibo/IWeiboResult.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Services.Weibo
+{
+    /// <summary>
+    /// Any kind of weibo object.
+    /// </summary>
+    public interface IWeiboResult
```

Comment:
Does it makes sense to have this public empty interface? The only place where this is used is https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/2515/files#diff-e484bad6b3c39a9d9efeb70810cf8edcR18 which is private, so just use the concrete implementation or just use object... It just feels weird to have an empty public interface.